### DISCUSSION TITLE
Pin multiple images in a single session

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#65] [Ideas]
https://twitter.com/ctisus/status/1486412362785644556

In the example given above, there are 4 images. Currently I have to hover over each image separately and select the pin option to get those images into the wiki. Is it possible to pin all the 4 images in a single session?

**Comment by arunnbabu81** [comment]
Is this possible with tiddlyclip

**Comment by buggyj** [comment]
I will think about it

**Reply by arunnbabu81** [reply]
Any updates regarding this

**Reply by arunnbabu81** [reply]
> I will think about it

Any new info regarding this feature request

**Comment by buggyj** [comment]
you could snap the html and then extract the images using a regex

**Reply by arunnbabu81** [reply]
> you could snap the html and then extract the images using a regex
> 

How to do the extraction using regex ?

**Reply by arunnbabu81** [reply]
> you could snap the html 
>
Did you meant snap or html clipping

**Reply by buggyj** [reply]
html clipping
